apps/wallet/src/ui/app/components/menu/content/index.tsx

```diff
@@ -52,17 +48,11 @@ function MenuContent() {
 			<ErrorBoundary>
 				<MainLocationContext.Provider value={mainLocation}>
 					<Routes location={menuUrl || ''}>
-						<Route path="/" element={<MenuList />} />
-						<Route path="/accounts" element={<AccountsSettings />}>
-							<Route path="connect-ledger-modal" element={<ConnectLedgerModalContainer />} />
-						</Route>
-						<Route path="/export/:account" element={<ExportAccount />} />
-						<Route path="/import-private-key" element={<ImportPrivateKey />} />
+						<Route path="/" element={<WalletSettingsMenuList />} />
```

Comment:
It seems now `MenuList` is not used anymore. Can we remove? Is it being used in other places? Any other components that we can remove?